crates/rslint_core/src/groups/errors/mod.rs

```diff
@@ -32,4 +32,5 @@ group! {
     valid_typeof::ValidTypeof,
     no_extra_boolean_cast::NoExtraBooleanCast,
     no_confusing_arrow::NoConfusingArrow,
+    no_dupe_class_members::NoDupeClassMembers
```

Comment:
Now that I see the list of rules here, I think it would be pretty nice to sort them in alphabetic order.